### PR TITLE
SG-22801 ShotGrid Update

### PR DESCRIPTION
**Pull request by eshokrgozar**
- Replaced instance of Shotgun with ShotGrid. 
- Updated the developer site links.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -258,12 +258,12 @@ shotgunEvents conf file::
 
     # A comma delimited list of paths where the framework should look for plugins to
     # load.
-    paths: /path/to/sg_jira_bridge/triggers, /path/to/any/other/shotgun/plugins
+    paths: /path/to/sg_jira_bridge/triggers, /path/to/any/other/shotgunEvents/plugins
```
Comment (jfboismenu) [inline docs/quickstart.rst:261]: Nice catch.

**Review by jfboismenu — APPROVED** [review]
Nice job. Must have been painful to get all those strings correct. :(

**Review by mathurf — CHANGES_REQUESTED** [review]
I think this line should be change to have ShotGrid instead of Shotgun
https://github.com/shotgunsoftware/sg-jira-bridge/blob/SG-22801_shotgrid_update/sg_jira/jira_session.py#L199

**Comment by eshokrgozar** [comment]
> I think this line should be change to have ShotGrid instead of Shotgun
> https://github.com/shotgunsoftware/sg-jira-bridge/blob/SG-22801_shotgrid_update/sg_jira/jira_session.py#L199

I only updated the documentation for this PR. We generate documentation that pulls the docstrings, so I updated those as well. But I've left everything else in the python file alone for now because we don't have time to re-test the tool.

**Review by mathurf — APPROVED** [review]
LGTM!
:shipit:

**Pull request merged**